PROVESFlightControllerReference/Components/ModeManager/ModeManager.cpp

```
@@ -46,6 +48,8 @@ void ModeManager ::init(FwSizeType queueDepth, FwEnumStoreType instance) {
 void ModeManager ::run_handler(FwIndexType portNum, U32 context) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'run_handler' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)